src/SingleProjectOverview.js

```diff
@@ -1,35 +1,85 @@
 import React, {Component} from 'react';
-import {Header, Button, Image, Progress, Statistic, Grid} from "semantic-ui-react";
+import {
+    Header,
+    Button,
+    Image,
+    Progress,
+    Statistic,
+    Grid,
+    Dimmer,
+    Form,
+    Select,
+    Input,
+    Radio, TextArea, Checkbox, Container
+} from "semantic-ui-react";
 import "./SingleProjectOverview.css";
 
 const options = [
```

Comment:
Deze onder de class definitie zetten of in de render zodat het alleen bereikbaar is op de plek waar het nodig is